### PR TITLE
Centre the media controls (and simplify some CSS along the way)

### PR DESCRIPTION
**Pull request by JeffersonBledsoe**
This PR centres the media controls (play, next, shuffle, etc.) as it is currently slightly off-centre. I simplified the CSS a little along the way

### CODE REVIEW

File: src/layouts/default/PlayerOSD/Player.vue [hunk 1]
```diff
@@ -1,8 +1,5 @@
 <template>
-  <div
-    :class="`${useFloatingPlayer ? 'mediacontrols-bg-2' : 'mediacontrols-bg-1'}`"
-    :style="`background: ${backgroundColor};`"
-  ></div>
+  <div class="mediacontrols-bg" :data-floating="useFloatingPlayer"></div>
```
Comment (marcelveldt) [inline src/layouts/default/PlayerOSD/Player.vue:2]: data-floating is a bit of a weird property name

File: src/layouts/default/PlayerOSD/Player.vue [hunk 2]
```diff
@@ -14,25 +11,16 @@
...
   />
 
-  <div
-    class="mediacontrols"
-    :style="`padding: ${
-      getBreakpointValue({ breakpoint: 'phone' }) ? 3 : 10
-    }px ${getBreakpointValue({ breakpoint: 'phone' }) ? 10 : 10}px;`"
-  >
-    <div :class="`mediacontrols-left-${getBreakpointValue('bp3') ? '1' : '2'}`">
+  <div class="mediacontrols" :data-mobile="mobile">
```
Comment (marcelveldt) [inline src/layouts/default/PlayerOSD/Player.vue:14]: see my previous comment: whats up with the "data-" prefixed value ?

File: src/layouts/default/PlayerOSD/Player.vue [hunk 3]
```diff
@@ -134,6 +123,8 @@ defineProps<Props>();
 
 // global refs
 const theme = useTheme();
+// Custom breakpoint for compatibility with `getBreakpointValue`. Can replace once we switch to using built-in Vuetify breakpoints
```
Comment (marcelveldt) [inline src/layouts/default/PlayerOSD/Player.vue:126]: Oh yeah, I looked into this a couple of weeks ago and its a complete mystery to me why we aren't just using default vuetify for this. My best guess is that this was not yet available back then in the vuetify beta's.

To me, we should now go (back) to using as much native vuetify as possible and just follow along

File: src/layouts/default/PlayerOSD/Player.vue (hunk 1 above)
Comment (JeffersonBledsoe) [reply src/layouts/default/PlayerOSD/Player.vue:2]: I'm using the `data-*` to reflect the state in the HTML ([Cube CSS](https://cube.fyi/exception.html#why-data-attributes) does the same thing). We could use a class here and do something like `class="mediacontrols-bg mediacontrols-bg__floating"`, but I find the data attribute a little clearer.

File: src/layouts/default/PlayerOSD/Player.vue (hunk 2 above)
Comment (JeffersonBledsoe) [reply src/layouts/default/PlayerOSD/Player.vue:14]: Replied in the previous comment 🙂

File: src/layouts/default/PlayerOSD/Player.vue (hunk 3 above)
Comment (JeffersonBledsoe) [reply src/layouts/default/PlayerOSD/Player.vue:126]: Agreed. I did try going straight to using the vuetify config with this change, but the breakpoints are out-of-sync for some things, causing some buttons to be display and others not at certain widths. We should do one single change to all of them at the same time to move to the Vuetify config to better catch any bugs that might crop up. I've slightly started this work in https://github.com/JeffersonBledsoe/music-assistant-frontend/tree/get-breakpoint-tidy-1

**Review by marcelveldt — APPROVED** [review]
Nice!

**Pull request merged**